### PR TITLE
Toggle Notes

### LINKED ISSUE #7: Add checkbox to show/hide note view

### PR DESCRIPTION
**Pull request by curran**
Will close #7

### CODE REVIEW

**Comment by curran** [comment]
We have a new checkbox!

image
On.

image
Off.

File: js/extremeNotesUI.js [hunk 1]
```diff
@@ -16,6 +16,19 @@ function ExtremeNotesUI(){
...
+                dispatch.call("extremes", this, checked);
+            } else {
+                // Initial value is true.
+                return true;
+            }
+        });
+
+    } // my() - Main Function Object
+
+    function checkbox(form, labelText, accessor){
```
Comment (seemantk) [inline js/extremeNotesUI.js:42]: Yes! generalizing++

File: js/extremeNotesUI.js [hunk 2]
```diff
@@ -16,6 +17,16 @@ function ExtremeNotesUI(){
...
+        // Checkbox for showing/hiding all notes.
+        checkbox(form, labelTextNotes, function (checked){
+            if(arguments.length){
+                dispatch.call("notes", this, checked);
+            } else {
+                // Initial value is true.
+                return true;
+            }
+        });
+
```
Comment (seemantk) [inline js/extremeNotesUI.js:29]: Brilliant!

**Review by seemantk — APPROVED** [review]

**Comment by seemantk** [comment]
Nice job, @curran. Looks fantastic!

**Comment by curran** [comment]
Thanks!

**Pull request merged**